JustPeek/Classes/PeekController.swift

```diff
@@ -50,8 +50,10 @@ internal protocol PeekHandler {
     fileprivate var peekHandler: PeekHandler?
     fileprivate var sourceViewController: UIViewController?
     
+    public var nativeForceTouchDisabled = false
```

Comment:
This is for using PeekReplacementHandler by force. I think that some users might be want it.